polkadot-parachains/integritee-runtime/Cargo.toml

```diff
@@ -37,6 +37,8 @@ frame-system-rpc-runtime-api = { default-features = false, git = "https://github
 # pallets
 pallet-aura = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.37" }
 pallet-balances = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.37" }
+pallet-bounties = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.37" }
+pallet-child-bounties = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.37" }
 pallet-collective = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.37" }
```

Comment:
std enablement missing

Reply:
Had no idea the CI would pass without the std checks. Good thing this was pointed out for the future

Reply:
that's why CI failed I guess